sbt-tzdb/src/main/resources/jvm/TzdbZoneRulesProvider.scala

```diff
@@ -1,7 +1,6 @@
 package
 
 import org.threeten.bp.DateTimeException
-import java.util.TreeMap
```

Comment:
The reason to use `TreeMap` in jvm is to avoid a half baked version (which is ZoneMap). Would it make more sense to have a native specific version?

Reply:
OK, I've reworked the PR, please have another look :+1: